Objects/complexobject.c

```diff
@@ -143,6 +189,64 @@ _Py_c_quot(Py_complex a, Py_complex b)
 
     return r;
 }
+
+Py_complex
+_Py_cr_quot(Py_complex a, double b)
+{
+    Py_complex r = a;
+    if (b) {
+        r.real /= b;
+        r.imag /= b;
+    }
+    else {
+        errno = EDOM;
+        r.real = r.imag = 0.0;
+    }
+    return r;
+}
+
+/* an equivalent of _Py_c_quot() function, when 1st argument is real */
+Py_complex
+_Py_rc_quot(double a, Py_complex b)
+{
+    Py_complex r;
+    const double abs_breal = b.real < 0 ? -b.real : b.real;
```

Comment:
Are the `const` useful (i.e. do they lead to better assembly?) or are they just to avoid redefining them by mistake?

Reply:
1. less differences with ``_Py_c_quot()``
2. yes, usually it's a good idea to mark a constant as a constant.  Just in case.